calculator/src/test/java/com/market/banica/calculator/service/ProductServiceImplTest.java

```diff
@@ -1,326 +1,327 @@
-        //Arrange
-        when(productBase.getDatabase()).thenReturn(demoDataBase);
-
-        //Act
-        productService.deleteProductFromDatabase(BANICA);
-
-        //Assert
-        verify(auroraClientSideService, times(1)).cancelSubscription(BANICA);
-        verify(backUpService, times(1)).writeBackUp();
-    }
-
-    @Test
-    void deleteProductFromParentIngredientsShouldThrowIllegalArgumentExceptionIfParentProductDoesNotExist() {
-        //Arrange
-        when(productBase.getDatabase()).thenReturn(new ConcurrentHashMap<>());
-        //Act//Assert
-        assertThrows(IllegalArgumentException.class, () -> productService.deleteProductFromParentIngredients(BANICA, PUMPKIN));
-    }
-
-    @Test
-    void deleteProductFromParentIngredientsShouldThrowIllegalArgumentExceptionIfParentProductDoesNotContainIngredient() {
-        //Arrange
-        when(productBase.getDatabase()).thenReturn(demoDataBase);
-        //Act//Assert
-        assertThrows(IllegalArgumentException.class, () -> productService.deleteProductFromParentIngredients(BANICA, PUMPKIN));
-    }
-
-    @Test
-    void deleteProductFromParentIngredientsShouldRemoveIngredientFromParentProductIngredients() {
-        //Arrange
-        setBanicaIngredients();
-        when(productBase.getDatabase()).thenReturn(demoDataBase);
-        //Act
-        productService.deleteProductFromParentIngredients(BANICA, PUMPKIN);
-        //Assert
-        verify(backUpService, times(1)).writeBackUp();
-        verifyNoMoreInteractions(backUpService, backUpService, auroraClientSideService);
-    }
-
-    @Test
-    void getProductAsListProductShouldReturnProductAndHisIngredients() {
-        //Arrange
-        setBanicaIngredients();
-        when(productBase.getDatabase()).thenReturn(demoDataBase);
-
-        //Act
-        List<Product> products = productService.getProductAsListProduct(BANICA);
-
-        //Assert
-        assertEquals(products.get(0), banica);
-        assertEquals(products.get(1), pumpkin);
-
-    }
-
-    @Test
-    void getProductAsListProductShouldReturnSingleProductAndWithoutIngredients() {
-        //Arrange
-        when(productBase.getDatabase()).thenReturn(demoDataBase);
-
-        //Act
-        List<Product> products = productService.getProductAsListProduct(BANICA);
-
-        //Assert
-        assertEquals(products.get(0), banica);
-
-    }
-
-    private void setBanicaIngredients() {
-        demoDataBase.put(PUMPKIN, pumpkin);
-        Map<String, Integer> ingredients = new ConcurrentHashMap<>();
-        ingredients.put(PUMPKIN, QUANTITY);
-        banica.setIngredients(ingredients);
-    }
-
-    private Map<String, Product> getDatabaseWithIngredients() {
-        Map<String, Product> ingredients = new ConcurrentHashMap<>();
-        ingredients.put(PUMPKIN, createProduct(PUMPKIN));
-        return ingredients;
-    }
-}
\ No newline at end of file
+//package com.market.banica.calculator.service;
```

Comment:
Is there a reason for the tests to be commented ?

Reply:
Fixed